### DISCUSSION TITLE
PicGo 相关中删除图片无效

### DISCUSSION THREAD

**Discussion by y761350477** [Molunerfinn/PicGo#691] [Q&A]
在相册中删除图片后，在 oss-browser 管理器中查看，图片依然存在的问题。
Bucket 已经赋权读写

![](https://y9287.oss-cn-shanghai.aliyuncs.com/2021-06-30/20210630105210-3e696a4-20210630105211.png)

**Comment by Molunerfinn** [comment]
PicGo没有支持远程删除图片。https://github.com/Molunerfinn/PicGo/blob/dev/FAQ.md
